File: java-webshop/src/main/java/com/adam/buzas/webshop/main/model/ArchiveLog.java
package com.adam.buzas.webshop.main.model;

import jakarta.persistence.*;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Entity
@Data
@NoArgsConstructor
@Table(name = "archive_log")
public class ArchiveLog {

    @Id
    private Integer id;

    private Integer archiveYear;

    private String status;

    private Integer archivedCount;

    private LocalDateTime startTime;

    private LocalDateTime endTime;
}

File: angular-webshop/src/app/models/archiveLog.ts
/**
 * api kérésnél visszakapott ArchiveLog-t reprezentálja, a szükséges adattagokkal
 */
export interface ArchiveLog{
  id: number;
  archiveYear: number;
  status: string;
  archivedCount: number; 
  startTime: string;
  endTime: string;
}
